From: Kevin Roy <kiniou@gmail.com>
Date: Thu, 14 Apr 2011 14:55:23 +0200
Subject: blender_thumbnailer

* Move thumbnailer script to an appropriate destination
* Change to python 3.2 since lintian is messing around with that
---
 release/bin/blender-thumbnailer.py |    4 ++--
 source/creator/CMakeLists.txt      |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/release/bin/blender-thumbnailer.py b/release/bin/blender-thumbnailer.py
index 8b93eeb..a056852 100755
--- a/release/bin/blender-thumbnailer.py
+++ b/release/bin/blender-thumbnailer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3.2
 
 # ##### BEGIN GPL LICENSE BLOCK #####
 #
@@ -24,7 +24,7 @@
 Thumbnailer runs with python 2.6 and 3.x.
 To run automatically with nautilus:
    gconftool --type boolean --set /desktop/gnome/thumbnailers/application@x-blender/enable true
-   gconftool --type string --set /desktop/gnome/thumbnailers/application@x-blender/command "blender-thumbnailer.py %u %o"
+   gconftool --type string --set /desktop/gnome/thumbnailers/application@x-blender/command "/usr/lib/blender/scripts/blender-thumbnailer.py %i %o"
 """
 
 import struct
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 79a7f5b..27c13d7 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -360,7 +360,7 @@ if(UNIX AND NOT APPLE)
 		)
 		install(
 			PROGRAMS ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/blender/scripts
 		)
 		install(
 			FILES ${CMAKE_SOURCE_DIR}/doc/manpage/blender.1
-- 
